fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/AvroDimensionRowParser.java

```diff
@@ -69,29 +62,22 @@ public AvroDimensionRowParser(DimensionFieldNameMapper dimensionFieldNameMapper,
      * </code></pre>
```

Comment:
The JSON listed is still how the avro schema is represented internally in the avro file, so it is still useful to have as a comment.